tests/unit/test_response_class_resolution.py

```diff
@@ -52,10 +52,10 @@ def handler(self) -> None:
     if app_response_class:
         app.response_class = app_response_class
 
-    route_handler, _ = app.routes[0].route_handler_map[HttpMethod.GET]  # type: ignore
+    route_handler = app.route_handler_method_map["/"][HttpMethod.GET]
 
     if handler_response_class:
-        route_handler.response_class = handler_response_class
+        route_handler.response_class = handler_response_class  # type: ignore
 
-    response_class = route_handler.resolve_response_class()
+    response_class = route_handler.resolve_response_class()  # type: ignore
```

Comment:
We try not to use blanket `type: ignore`s. Can you add the specific error codes?

Reply:
Actually, we should enforce this. There's a mypy setting `enable_error_code = "ignore-without-code"` that does this. Would you be interested in creating a PR for this? :)